mycroft/client/speech/mic.py

```diff
@@ -356,14 +362,16 @@ def _upload_wake_word(self, audio):
         else:
             model_hash = '0'
 
-        metadata = {
+        return {
             'name': self.wake_word_name.replace(' ', '-'),
             'engine': md5(ww_module.encode('utf-8')).hexdigest(),
             'time': str(int(1000 * get_time())),
             'sessionId': SessionManager.get().session_id,
             'accountId': self.account_id,
             'model': str(model_hash)
         }
```

Comment:
I wonder if you want a simple list here. This currently results in the filename `name.engine.time.sessionId.accountId.model.wav` (join of a dict seem to be joining the keys).

Alternatively you could use the `.values()` but I'm not sure if that will always preserve the order so maybe make it an ordered dict?

Reply:
Works better if I hit "Save" in my editor...

I'd already changed that to use values().  I don't honestly care that much about order when reviewing a local copy, so I don't think it matters much

Reply:
I personally care (otherwise I wouldn't have mentioned it). It's good style to guarantee consistency and if anyone makes a script for doing any type of analysis on the recorded wakewords or something it'd be good if it can easily parse out the name.

Reply:
Fair enough!